CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # AWS Tamr Config Repo
 * Adds variable `emr_cluster_name_prefix` that is passed to the config `TAMR_DATASET_EMR_CLUSTER_NAME_PREFIX` in the file
```

Comment:
Which version is this change associated with?

Reply:
ah, I was wondering why the changelog was so weird with that repeated statement. I totally missed this line above it.


I was wondering why the 2.5.0 and 2.4.5 had the same change, and it's because this line should have been here.